apps/accounts/risk_handlers.py

```diff
@@ -174,10 +188,10 @@ def handle_change_password_add(self):
             'check_conn_after_change': True,
             "name": "Push account password: {}@{}".format(self.username, asset.name),
         }
-        execution.save()
-        execution.start()
+        self.start_execution(execution)
 
-        GatheredAccount.objects.filter(asset=self.asset, username=self.username).update(
-            present=True
+        (
+            GatheredAccount.objects
+            .filter(asset=self.asset, username=self.username)
+            .update(present=True)
         )
-        return execution.summary
```

Comment:
There are no obvious issues nor irregularities with the code provided. However, there is room for further documentation since it doesn't include error handling which should be added to handle exceptions.

Moreover, consider adding type checks in order to enhance readability of the code. It's important to note that certain functions such as `handle_change_password` don't seem appropriate because they modify database entries without checking what happened and whether an issue occurred at all when modifying accounts. Please review if necessary before proceeding.